src/components/content/common/ocl/DeploymentScript.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Button, Popover } from 'antd';
+import React from 'react';
+import YAML from 'yaml';
+import oclDisplayStyles from '../../../../styles/ocl-display.module.css';
+import { Deployment } from '../../../../xpanse-api/generated';
+
+export function DeploymentScriptText({ deployment }: { deployment: Deployment }): React.JSX.Element {
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    if (deployment) {
+        const yamlDocument = new YAML.Document();
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-expect-error
+        yamlDocument.contents = deployment.deployer;
```

Comment:
here we must use the content from either deployer or the ScriptsRepo

Reply:
![image](https://github.com/user-attachments/assets/c2c0899a-a7cf-499c-a02d-778279ca9744)

fixed.